oak_idl/src/lib.rs

```diff
@@ -125,97 +129,117 @@ impl Transport for Channel {
     }
 }
 
+/// A helper struct to facilitate building a [`Message`].
+///
+/// It delegates most methods to the underlying [`flatbuffers::FlatBufferBuilder`] instance, but it
+/// adds a [`MessageBuilder::finish`] method that returns a completed [`Message`] instance.
```

Comment:
It would be useful to document the purpose of this as part of this comment. 

Is this help meant as a generalized improvements to the ergonomics of flatbuffer on rust? If so it might be helpful to move it to a seperate helpers file that's non Oak IDL specific